Directory.Packages.props

```diff
@@ -1,20 +1,21 @@
-<Project>
+<Project>
+
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
+    <!-- Using multiple feeds isn't supported by Maestro: https://github.com/dotnet/arcade/issues/14155. -->
+    <NoWarn>$(NoWarn);NU1507</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageVersion Include="Microsoft.DotNet.RemoteExecutor" Version="$(MicrosoftDotNetRemoteExecutorVersion)" />
+    <PackageVersion Include="Microsoft.DotNet.XUnitExtensions" Version="$(MicrosoftDotNetXUnitExtensionsVersion)" />
+    <PackageVersion Include="System.Numerics.Vectors" Version="$(SystemNumericsVectorsVersion)" />
+    <PackageVersion Include="System.Runtime.CompilerServices.Unsafe" Version="$(SystemRuntimeCompilerServicesUnsafeVersion)" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <GlobalPackageReference Include="Microsoft.DotNet.ApiCompat.Task" Version="9.0.100-alpha.1.23560.11" />
+  </ItemGroup>

```

Comment:
This diff looks wrong. GH says that all these lines got changed but I don't see any actual changes.

Reply:
Whitespace changes. I modified the file with Visual Studio on Windows. Did you create the file in Unix by any chance?
Maybe we're missing a line in .editorconfig specifying the default whitespace we need to use.
For now you can just ignore the whitespace changes: 
![image](https://github.com/dotnet/maintenance-packages/assets/1175054/646f45d5-ed24-4fb8-8a5e-bb0881645a17)